tests/phpunit/tests/privacy/wpPrivacyGeneratePersonalDataExportFile.php

```diff
@@ -71,7 +71,7 @@ public function set_up() {
 		$this->export_file_name = '';
 
 		if ( ! $this->remove_exports_dir() ) {
-			$this->markTestSkipped( 'Existing exports directory could not be removed. Skipping test.' );
+			$this->fail( 'Existing exports directory could not be removed. Skipping test.' );
```

Comment:
```suggestion
			$this->fail( 'Existing exports directory could not be removed.' );
```

Though, again, I don't think failing the test is correct here. There may be contributor system specific reasons why the directory could not be removed.